server.py

```diff
@@ -85,31 +91,33 @@ def process_slack_command(db):
     # make our lives a little easier
     slack_data = DictObject(request.forms)
 
-    # match help text
-    if slack_data.text == 'help':
-        return {
-            "text": "Request this help text with `/xqzes help`\n"
-                    "Request an excuse (visible to everyone) with `/xqzes`\n"
-                    "Submit a new excuse to a moderator with "
-                    "`/xqzes add <your text here>`\n"
-                    "e.g: `/xqzes add I was shopping!`"
-        }
-    elif slack_data.text.startswith("add"):
-        # here we want to add a new non-approved excuse
-        excuse_text = slack_data.text.lstrip(" add ")
-        if len(excuse_text) > 140:
+    # small chance text is empty (if a stupid dev is curling manually)
+    if 'text' in slack_data.attributes.keys():
```

Comment:
indeed it turns out it is, i had to fiddle to get this to work, so eventually went with what worked. fixed.